### ISSUE TITLE
ITIS - Nychia marshalli subsp. bantu Hutchinson, 1929

### ISSUE THREAD

**Issue by ManonGros** [CatalogueOfLife/data#368]
**Describe the problem**:
Feedback originally provided by @lmoliner on the GBIF Feedback portal:
They think the correct name for Nychia marshalli subsp. bantu Hutchinson, 1929 should be Nychia limpida bantu.

I think this might be an issue for @DaveNicolson

**Link to effected CoL webpages**:
https://www.catalogueoflife.org/data/taxon/5JXFH

**Literature references**:
In the paper by Hutchinson (1929) A revision of the Notonectidae and Corixidae of South Africa (available on Biodiversity Heritage Library), it is described as Nychia limpida bantu n. sp., with records in the Caprivi Strip, Bechuanaland Protectorate (Botswana), and Northern Rhodesia (Zambia). Stal (1858) was the first who named the species as N. limpida, then it was synonymised by Scott (1872) to Antipalocoris marshalli, later to Nychia marshalli by Kirkaldy (1901).

**Comment by yroskov** [comment]
@DaveNicolson, it's for you

**Comment by DaveNicolson** [comment]
We will have a look. ITIS cites a 1966 catalog for the subspecies name & its status, and the 1995 catalog from Aukema et al. for the species names (Nychia marshalli & N. limpida) so we would presume those supersede those earlier works that were cited... Maybe the 1966 catalog (cited for subspecies here) missed those earlier works. Let's see what comes of it...

**Comment by DaveNicolson** [comment]
On the face of it, something looks wrong with this suggestion, which seems to indicate that a junior name should be treated as a senior name... I will re-order the submitted sentences to show what I mean...

AFFECTED NAMES IN ITIS NOW (all valid/accepted now):
Nychia limpida Stål, 1859
Nychia marshalli (Scott in Marshall, 1872)
  Nychia marshalli bantu Hutchinson, 1929
  Nychia marshalli horvathi Lundblad, 1933
  Nychia marshalli marshalli (Scott in Marshall, 1872)

Historic background from the submitter:
"Stal (1858) was the first who named the species as N. limpida, then it was synonymised by Scott (1872) to Antipalocoris marshalli [dates to 1879, so junior!], later to Nychia marshalli by Kirkaldy (1901)."

This sounds backwards, since limpida is senior to marshalli....??

More specifically...:
"In the paper by Hutchinson (1929) A revision of the Notonectidae and Corixidae of South Africa (available on Biodiversity Heritage Library), it is described as Nychia limpida bantu n. sp., with records in the Caprivi Strip, Bechuanaland Protectorate (Botswana), and Northern Rhodesia (Zambia)."

And the submitter's summary:
"I think the correct name for this species [sic... the name the comment is linked to is a subspecies, Nychia marshalli bantu Hutchinson, 1929] should be Nychia limpida bantu and not Nychia marshalli bantu."

Now... If this were a question of to which species the subspecies in question belongs, then that would be one thing. But the submitter seems to be suggesting that Nychia limpida Stål, 1859 should be synonymized under the junior name Nychia marshalli (Scott in Marshall, 1872). Unless there is something that makes the older name unavailable or otherwise unusable (e.g., homonymy), this would not make sense... I'm trying to figure out how to ask someone to look into this without wasting their time... Sigh...

**Comment by DaveNicolson** [comment]
J.T. Polhemus' section on the Notonectidae in the 1995 vol. 1 of the Catalogue of the Heteroptera of the Palaearctic Region (Aukema & Reiger, eds.) clearly treats both species as separate & valid, with Nychia limpida Stål, 1859 distributed in China, and Nychia marshalli (Scott in Marshall, 1872) distributed in France (Corsica) and Tropical Africa. The idea that subspecies bantu (of Africa) should be moved from the African species to the Chinese species doesn't make sense, to me.
<img width="889" alt="Nychia_per_Polhemus1995" src="https://user-images.githubusercontent.com/74622398/145248749-c3cdede9-475e-426d-b1a2-57f2f7ccfc4d.png">

IF the two species were to be synonymized, the senior name would be used, so it would be Nychia limpida, and the subspecies would then be N. limpida bantu, but it looks to me like the modern literature keep them separate. Therefore, unless new information is brought to light, the treatment in ITIS appears to be correct.

**Comment by lmoliner** [comment]
Dear Dave,

Thank you very much for your detailed reply, I am doing a PhD in southern African macroinvertebrates and I think this genus (and many others) needs revision. There are records of Nychia limpida in South Africa (Cape, Transvaal), Zambia, and Zimbabwe, as well as Europe (Corsica) so I strongly disagree with Polhemus.

Besides, his catalogue is for the Palaearctic Region (therefore, doesn't include southern Africa, and he even didn't get the dates right, as Stal described the genus Nychia in 1858, and not 1860...).

To me is really odd that several websites (such as catalogue of life, and ITIS) cite the species as Nychia marshalli bantu (Hutchinson, 1929) when is clear, on page 411 (please, see a screenshot of his book "A Revision of the Notonectidae and Corixidae of South Africa") that he named the species as Nychia limpida bantu n. subp.

In conclusion, and until someone revises the genus, it seems to me that Nychia limpida should prevail as original name, and N. marshalli and Antipalocoris marshalli are synonyms.

Regards,

Luis Moliner Cachazo
PhD Student at King's College London

[cid:46f749bc-dae0-4345-9411-10463b260959]
[cid:88e1054b-0482-4f98-90b3-fe9016dac5b5]



________________________________

**Comment by DaveNicolson** [comment]
Thanks for this additional information, Luis. I can heartily agree that a revision of the group would be extremely helpful for cases like this. I'm not a specialist in this group, but for ITIS we have tried to follow recent revisions or catalogs/checklists when available, and where they are not we fall back on regional catalogs & older revisions, so for groups like this there will inevitably be some cases where there remain problems that can't be resolved without such revisions.

It's fairly clear that in the early 1900s many taxonomists treated these two species as synonymous (Lundblad's 1933(?) work speaks at length about it, although I can't read his German very well, and it seems like there is a certain amount of chresonymy in his discussion, that is, use of names not always in a way that includes the types, but rather as used in other works with various taxonomic views), in which Nychia limpida would be the name to use (with marshalli as synonym), and Hutchinson took this view (see [p. 409](https://www.biodiversitylibrary.org/page/40656639)), so it is not surprising in the least that he would name his subspecies in Nychia limpida.

In Poisson's 1966 catalog of African & Malagasy Notonectidae (don't think it's online, we have a poor scanned PDF), he mentioned limpida (as type species for Nychia), and listed the species N. marshalli (in 3 parts, typical, bantu and horvathi), suggesting he viewed the two species as distinct (otherwise he would have had to use limpida instead of marshalli, due to priority). That, taken with the 1995 palaearctic catalog (which listed the 2 species separately, as already discussed), and some other works, was how we ended up with what is now in ITIS and COL for this genus. I do hope someone will eventually take this up and make a global revision and resolve this (at least for a time) and other similar groups... For now, I think we have to wait for additional workers to tackle it in one way or another.

As to the dates, in our experience, it is not at all uncommon for the dates used to vary, sometimes based on detailed examinations of the volumes, wrappers, signatures, etc., vs. just listing the year on the title page of the volume (etc.). In ITIS we ended up using the year 1859 for limpida, although I don't have notes in front of me as to why that was the choice...

Best,
Dave

**Comment by lmoliner** [comment]
Hi Dave,

Thank you so much for this interesting discussion, I do appreciate the time you have spent looking for references and examples, and your very detailed answers. I'm working on a database of freshwater species in the Okavango Delta that I hope to get publish at some point, and there are some specimens of Nychia that have been found there, so all these details are really useful.

Kind regards,

Luis

________________________________

**Comment by DaveNicolson** [comment]
Hi Luis,

As a follow-up in the year of publication being 1858, 1859 or 1860, as you've noted, the Polhemus work used 1860. The work he cites is here (on the page for this name, work was pp. 219-298... this is the same work cited by Hutchinson):
https://www.biodiversitylibrary.org/page/2473891

And here it is indicated that the work was from 1859:
https://www.biodiversitylibrary.org/page/2473832

It looks to me like 1859 is the best available date for that work. I don't know why Polhemus ended up with 1860 ;-)

Best,
Dave

**Comment by lmoliner** [comment]
Excellent, thank you so much for the links and replies!
Best wishes,
Luis

________________________________
